### DISCUSSION TITLE
一个关于datagrid的需求

### DISCUSSION THREAD

**Discussion by 158545614** [BYJRK/DotNet-Discussions#345] [WPF 开发]
大佬，请教一个问题，我有一张datatable，datatable 直接绑定到datagrid的itemsource，并且自动生成列，datatable 有4列  Max  Min Result  State  ，Max  Min为上下限 ，我希望我在result列里输入值之后，如果值在上下限之间，则State单元格显示PASS  并且变成绿色 如果不在上下限之间，显示fail  变成红色，如果result没有输入，则state单元格不显示任何字符，并显示为白色。这个需求用触发器要怎么实现，或者是不是用行为更合适，谢谢

**Comment by 158545614** [comment]
有没有大佬给个思路  谢谢

**Reply by designerlast** [reply]
参考这个代码，使用多重绑定，绑定需要对比的结果，max，min数值，在通过Convert来转化成bool,来做结果的颜色改变
![image](https://github.com/user-attachments/assets/de4111ef-9973-4b0c-b3f9-ef8c79370c65)
